### DISCUSSION TITLE
Display job summaries in PR checks view

### DISCUSSION THREAD

**Discussion by alvaromartmart** [community/community#16451] [Actions]
The new job summaries feature for Github Actions is great for providing a more visual feedback on workflow results.

However, it would be great if the latest summary/summaries could be also viewed directly from the PR status checks view. That view is already displaying the status of all checks so for me it would be natural to either as a (collapsable?) section under each check, like the mockup below:

![Screenshot 2022-05-12 at 16 49 09](https://user-images.githubusercontent.com/18287793/168103410-755f93cc-7f4e-4fb8-bab4-35da2cf52e13.png)

... or aggregated inside a "show checks summary" section, though it might be less readable

**Comment by saveman71** [comment]
The issue currently is that even if you add a summary, discoverability of it is very poor and other developers won't necessarily have the reflex to go to the summaries to see if their code is good or bad, as long as the checks pass.

My use case would be a technical debt report that would be non blocking, and I wanted to avoid adding a comment (needs permissions) and summaries seemed the perfect fit.

**Comment by AmyJoT** [comment]
Would be nice for the option of the `details` link to direct to the summary page. That's something we are having to make workarounds for

**Comment by pirate** [comment]
yeah this feature would be amazing, honestly even just being able to put one line of short text would be better than not having any additional info inline other than `succeeded` / `failed` / `skipped`. 

cc @konradpabjan I feel like this would be an great improvement to your awesome work on markdown job summaries.

**Comment by pirate** [comment]
Also this discussion might be a duplicate: https://github.com/orgs/community/discussions/53528
